ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -231,9 +231,9 @@ def Inv(*, x, name="Inv"):
     return ivy.astype(ivy.reciprocal(x), x.dtype)
 
 
-@to_ivy_arrays_and_back
-def Reciprocal(*, x, name=None):
-    return ivy.reciprocal(x)
+Reciprocal = to_ivy_arrays_and_back(
```

Comment:
Any reason we are not using
```python
@to_ivy_arrays_and_back
def Reciprocal(...
```
format?

Reply:
It is mentioned here - https://github.com/unifyai/ivy/issues/1565 to keep an alias rather than creating a duplicate implementation.